app/src/sharedTest/java/org/oppia/app/activity/InputInteractionViewTestActivityTest.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.app.activity
+
+import android.app.Activity
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.model.InteractionObject
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.action.ViewActions.typeText
+import com.google.common.truth.Truth.assertThat
+import org.oppia.app.customview.interaction.NumericInputInteractionView
+
+/** Tests for [InputInteractionViewTestActivity]. */
+@RunWith(AndroidJUnit4::class)
+class InputInteractionViewTestActivityTest {
```

Comment:
Please also add 1 test verifying that a configuration change properly retains the text typed into the interaction view (and that its subsequent pending answer is correct).

Reply:
resolved